### PR TITLE
Smart Proxy Logs feature support

### PR DESCRIPTION
**Pull request by lzap**
Second attempt to add the Logs feature.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -322,6 +327,8 @@
   $templates                  = $foreman_proxy::params::templates,
   $templates_listen_on        = $foreman_proxy::params::templates_listen_on,
   $template_url               = $foreman_proxy::params::template_url,
+  $logs                       = $foreman_proxy::params::logs,
```
Comment (ekohl) [inline manifests/init.pp]: Mind validating these with ```validate_bool``` and ```validate_listen_on```?

File: manifests/params.pp [hunk 2]
```diff
@@ -173,6 +173,10 @@
   $templates_listen_on = 'both'
   $template_url        = "http://${::fqdn}:${http_port}"
 
+  # Logs settings
+  $templates           = true
```
Comment (ekohl) [inline manifests/params.pp]: This should be $logs

**Comment by lzap** [comment]
This is my copy*paste attempt. :-)

**Comment by lzap** [comment]
Sure, fixed all. Let's see what Jenkins says.

**Comment by lzap** [comment]
For the record, do not merge before https://github.com/theforeman/smart-proxy/pull/367

**Comment by mmoll** [comment]
looking good, but a test would be nice :mag:

File: manifests/params.pp [hunk 3]
```diff
@@ -173,6 +173,10 @@
   $templates_listen_on = 'both'
   $template_url        = "http://${::fqdn}:${http_port}"
 
+  # Logs settings
+  $logs           = true
```
Comment (domcleal) [inline manifests/params.pp]: should be `false` to match smart proxy default

File: manifests/init.pp (hunk 1 above)
Comment (ekohl) [inline manifests/init.pp]: (done)

**Comment by lzap** [comment]
The PR was merged. Can you show me how do we test those? I can't locate it. You don't want me to write puppet tests :-)

**Comment by domcleal** [comment]
spec/classes/foreman_proxy__config__spec.rb has examples for all of the existing config files, it should be added to there.

**Comment by lzap** [comment]
Added tests.

**Comment by mmoll** [comment]
tests fail...
```
        expected: ["---", ":enabled: true"]
             got: ["---", ":enabled: https"]
```

**Comment by lzap** [comment]
Yeah sorry, tests do not work locally for me. Pushed and fingers crossed.

File: manifests/params.pp (hunk 3 above)
Comment (lzap) [inline manifests/params.pp]: Changed as well.

**Comment by ekohl** [comment]
:+1: on green tests

**Comment by mmoll** [comment]
merged, díky @lzap!

**Pull request closed without merging**